### PR TITLE
[API] Get admin detail information

### PR DESCRIPTION
**Pull request by dawkaa**
I added it to get detailed admin information.

| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13           |
| Bug fix?        | no                                                       |
| New feature?    | yes                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no   |
| Related tickets | fixes #X, partially #Y, mentioned in #Z                      |
| License         | MIT                                                          |

### CODE REVIEW

File: src/Sylius/Bundle/ApiBundle/EventListener/AdminAuthenticationSuccessListener.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) https://github.com/dawkaa
```
Comment (NoResponseMate) [inline src/Sylius/Bundle/ApiBundle/EventListener/AdminAuthenticationSuccessListener.php]:
```suggestion
 * (c) Paweł Jędrzejewski
```

**Review by NoResponseMate — COMMENTED** [review]
Hi @dawkaa

Could you add a spec for the new listener?

File: src/Sylius/Bundle/ApiBundle/EventListener/AdminAuthenticationSuccessListener.php (hunk 1 above)
Comment (dawkaa) [reply src/Sylius/Bundle/ApiBundle/EventListener/AdminAuthenticationSuccessListener.php]: fixed it

**Comment by dawkaa** [comment]
> Hi @dawkaa
> 
> Could you add a spec for the new listener?

Hi @NoResponseMate, I added a spec for the new listener

**Comment by dawkaa** [comment]
@jakubtobiasz

**Comment by dawkaa** [comment]
@NoResponseMate

**Review by jakubtobiasz — APPROVED** [review]

**Review by NoResponseMate — APPROVED** [review]

**Review by coldic3 — APPROVED** [review]

**Comment by jakubtobiasz** [comment]
In the feature, please consider using git rebase, as such commits are unnecessary.
CleanShot 2023-02-02 at 15 40 24@2x

Anyway, thanks for your work!🙏🏼

**Comment by jakubtobiasz** [comment]
Thank you, @dawkaa!

**Comment by dawkaa** [comment]
> Thank you, @dawkaa!
:smile:

**Comment by dawkaa** [comment]
> In the feature, please consider using git rebase, as such commits are unnecessary. CleanShot 2023-02-02 at 15 40 24@2x
> 
> Anyway, thanks for your work!🙏🏼

Thanks, I see :bowtie:

**Pull request merged**